docs/operate/customize/collections.md

```diff
@@ -0,0 +1,179 @@
+_Introduced in v13_
+
+Collections provide a powerful way to curate and organize records within your InvenioRDM instance. They define sets of records based on search filters and can be organized hierarchically to create meaningful groupings of content.
+
+## Overview
+
+Collections are dynamic, query-based groupings of records that automatically include all records matching a specified search filter. They enable you to:
+
+- **Organize records thematically** - Group records by subject, resource type, funding program, or any metadata field
+- **Create hierarchical structures** - Build nested collections to guide users in browsing records
+- **Display curated content** - Present collections on dedicated pages within communities or globally
+
+Each collection is stored in the database with a search query string that dynamically fetches matching records. This approach ensures collections stay current as new records are added to your repository.
+
+!!! info "Architecture Details"
+    For technical implementation details, see [RFC0079](https://github.com/inveniosoftware/rfcs/blob/master/rfcs/rdm-0079-collections.md).
+
+## Configuration
+
+To enable displaying the communities "Browse" tab in your InvenioRDM instance, add to your `invenio.cfg`:
+
+```python
+COMMUNITIES_SHOW_BROWSE_MENU_ENTRY = True
+```
+
+## Managing Collections
+
+Collections are organized within **Collection Trees** - hierarchical structures that allow you to create logical groupings and nested relationships. Collection trees serve as the root containers for your collections and can be:
+
+- **Community-specific** - Scoped to records of a particular community
+- **Global** - Scoped across records of your entire instance
+
+!!! info "Global collections"
+    Global collections display is not yet implemented in InvenioRDM v13.
+
+Before creating collections, you need:
+
+1. **Access to Python shell** - Collections are currently managed via `invenio shell`
+2. **A community with sub-communities enabled** - Collections can be scoped to communities, so you need at least one community. That community must have `children.allow: true` set, which allows it to have sub-communities (and thus display the "Browse" tab).
```

Comment:
I would like to actually link to the sub-communities docs from here, which can explain how to configure a community with `children.allow: true`.